config.go

```diff
@@ -34,6 +43,11 @@ type Config[H Hash] struct {
 	// in current block can't be found in memory pool. The slice received by
 	// this callback MUST NOT be changed.
 	RequestTx func(h ...H)
+	// SubscribeForTxs is a callback which is called when dBFT needs to track incoming
+	// mempool transactions. Subscription supposed to be single-use, no unsubscription
+	// is initiated by dBFT hence it's the user's duty to manage and release resources.
+	// This callback is active iff MaxTimePerBlock is set.
+	SubscribeForTxs func()
```

Comment:
You can check that _both_ are set or unset somewhere.

Reply:
Agree.